test/utils/UtilsSpec.scala

```diff
@@ -24,20 +24,20 @@ import domain.StandingAuthorityFile
 import Utils._
 
```

Comment:
Could you please make use of constants  for the below test case to remove the magic number?

"return correct list of GB and XI authorities partitioned by the file name pattern"

You could use below vlas to replace the existing hard coded values